packages/admin/src/FinalForm.tsx

```diff
@@ -52,12 +54,13 @@ export function FinalForm<FormValues = AnyObject>(props: IProps<FormValues>) {
         />
     );
 
-    function RenderForm(formRenderProps: FormRenderProps<FormValues>) {
+    function RenderForm({ formContext = {}, ...formRenderProps }: FormRenderProps<FormValues> & { formContext: Partial<FinalFormContext> }) {
         const { mutators } = formRenderProps.form;
         const setFieldData = mutators.setFieldData as ((...args: any[]) => any) | undefined;
 
         const submit = React.useCallback(
             (event: any) => {
+                event.preventDefault(); //  Prevents from reloading the page with GET-params on submit
```

Comment:
Shouldn't final-form already take care of this?

Reply:
It's weird yes, it' happens in some cases in storybook (iframe-context).